### PR TITLE
Move language guide to `learn/what_is_meilisearch`

### LINKED ISSUE #1373: Move `reference/features/language` to `learn/what_is_meilisearch`
As part of our ongoing site restructure, I propose we move the Language guide to `learn/advanced_topics`. 

Possibly not the best place as I don't feel this document is particularly advanced. Happy to hear suggestions of other adequate spots.

### PR DESCRIPTION
**Pull request by guimachiavelli**
closes #1373

### CODE REVIEW

File: .vuepress/config.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ module.exports = {
             '/learn/what_is_meilisearch/comparison_to_alternatives',
             '/learn/what_is_meilisearch/telemetry',
             '/learn/what_is_meilisearch/contact',
+            '/learn/what_is_meilisearch/language',
```
Comment (maryamsulemani97) [inline .vuepress/config.js:54]: Can we have  '/learn/what_is_meilisearch/contact', at the bottom instead?

**Review by maryamsulemani97 — CHANGES_REQUESTED** [review]

File: .vuepress/config.js (hunk 1 above)
Comment (guimachiavelli) [reply .vuepress/config.js:54]: Sure, probably a good idea!

Comment (guimachiavelli) [inline .vuepress/config.js:54]:
```suggestion
            '/learn/what_is_meilisearch/language',
            '/learn/what_is_meilisearch/contact',
```

**Review by maryamsulemani97 — APPROVED** [review]
☃️

**Comment by guimachiavelli** [comment]
bors merge

**Pull request merged**